README.md

```diff
@@ -70,6 +71,19 @@ end
 
 More details on https://hexdocs.pm/tower/Tower.html#module-manual-reporting.
 
+### Configuration
+
+`TowerSentry` supports the following configuration options:
+
+- `:dsn` (`t:String.t/0`) - The DSN for your Sentry project. Setting this option is mandatory. Learn more about DSNs in the [official Sentry documentation](https://docs.sentry.io/concepts/key-terms/dsn-explainer/).
+- `:environment_name` (`t:String.t/0` or `t:atom/0`) - The current environment name. Setting this option is mandatory. Learn more about environments in the [official Sentry documentation](https://docs.sentry.io/concepts/key-terms/environments/).
+
+> #### Note on configuring the `:sentry` app directly {: .warning}
+>
+> `TowerSentry` currently depends on the [official Sentry SDK for Elixir](`e:sentry:readme.html`) for some internal functionality. It is however considered to be an implementation detail of `TowerSentry`.
+>
+> This means that while configuring the `:sentry` application directly _will_ work (outside of the options listed above, which `TowerSentry` overrides), you are doing so at your own risk; the `:sentry` dependency could be removed at any time in favor of a home grown implementation.
```

Comment:
I was debating whether to invite users to open issues or PRs for missing configuration options but maybe that's obvious enough?

Reply:
Yes, we OK not explicitly stating that.